selftests/unit/test_test.py

```diff
@@ -94,16 +90,10 @@ def check(uid, name, variant, exp_logdir):
         # Shrink variant
         check("a" * 253, "whatever", {"variant_id": 99}, "a" * 253 + "_9")
         check("a" * 254, "whatever", {"variant_id": 99}, "a" * 254 + "_")
-        # No variant
```

Comment:
It was removed because it was not needed. The last assertEqual  would be something like this:

```python
        self.assertEqual(os.path.basename(tst.workdir),
                                   os.path.basename(tst.workdir))
```

Which seems unnecessary to me.